inigomarquinez/how-to-develop-a-package-locally
# Feature/set up storybook

### Type:
* [ ] cicd: helm, docker & cicd adjustments.
* [x] feature: new capabilities.
* [ ] fix: bug, hotfix, etc.
* [ ] refactor: enhacements.
* [ ] style: changes in styles.
* [ ] other: docs, tests.

### What's the focus of this PR:
- Set up storybook

### How to review this PR:
- Check out the branch
- Execute `npm install`
- Execute `npm storybook`
